static/xref/filter/index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Search cross references</title>
+    <link rel="stylesheet" href="style.css?v=2020-06-03" />
```

Comment:
For shame!... but ok :) we could tell it to revalidate via HTTP, no?

Reply:
I was thinking of using immutable cache on static asset files through server.

Reply:
yeah, that could be nice.